maintainers/maintainer-list.nix

```diff
@@ -1736,7 +1736,7 @@
     name = "Chris Double";
   };
   dpaetzel = {
-    email = "david.a.paetzel@gmail.com";
+    email = "david.paetzel@posteo.de";
```

Comment:
this should be a separate commit, and probably a different PR

Reply:
OK, different PR then.